### PR TITLE
Included wiki_link and wiki_pageid in artist model

### PR DESCRIPTION
**Pull request by angelinrashmi2000**
Included wiki_link and wiki_pageid in artist model. This can be used by group 3 and 4 to extract relationships.

### CODE REVIEW

**Comment by Henni** [comment]
Why both `wiki_link` and `wiki_pageid`?
Isn't `wiki_pageid` a reference to a website entry which itself contains a url to the wiki page?

**Comment by ShilpaGhanashyamGore** [comment]
@Henni  : Wikipage id  is the unique Id that Wikipedia gives to each entry .It was decided in the seminar on last day to have it as well .Wiki Link is the url of wiki page .

**Review by ShilpaGhanashyamGore — APPROVED** [review]
Okay!

**Comment by Sandr0x00** [comment]
Since we do not work with the link, this may be interesting for @MusicConnectionMachine/group-2 .
I linked you, just for info.

**Comment by ShilpaGhanashyamGore** [comment]
@angelinrashmi2000 : There is a compilation error with this checkin .

File: api/models/artists.js [hunk 1]
```diff
@@ -38,6 +38,12 @@ module.exports = function (context) {
         source_link: {
             type: context.Sequelize.TEXT
         }
```
Comment (TimHenkelmann) [inline api/models/artists.js:40]: i think there is a "," missing here...

**Pull request merged**